### PR TITLE
Turn off online swagger validation by default by introducing a new string argument for setup_swagger function swagger_validator_url with default value ''. Documentation is updated.

### PR DESCRIPTION
**Pull request by olexandr-klymenko**
Turn off online swagger validation by default [#36](https://github.com/cr0hn/aiohttp-swagger/issues/36) by introducing a new string argument for **setup_swagger** function **swagger_validator_url** with default value **''**. Documentation is updated.

### CODE REVIEW

File: aiohttp_swagger/__init__.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ def setup_swagger(app: web.Application,
                   swagger_from_file: str = None,
                   swagger_url: str = "/api/doc",
                   api_base_url: str = "/",
+                  swagger_validator_url: str = '',
```
Comment (stasfilin) [inline aiohttp_swagger/__init__.py]: Change to 
```python
swagger_validator_url: str = "",
```
We need to use one style

Comment (olexandr-klymenko) [reply aiohttp_swagger/__init__.py]: should I create new PR?

Comment (stasfilin) [reply aiohttp_swagger/__init__.py]: Now, just commit and push changes

Comment (olexandr-klymenko) [reply aiohttp_swagger/__init__.py]: done

**Comment by olexandr-klymenko** [comment]
So, when is it going to be merged?

**Comment by alxpy** [comment]
@cr0hn can you merge this PR please?

**Comment by goranpavlovic** [comment]
is this still actual, if yes, have plan to merge this soon.

**Pull request merged**